src/components/MultiSelectGroup/MultiSelectGroup.mdx

```diff
@@ -102,6 +103,14 @@ argument value should be passed to the `width` prop of the `MultiSelect`.
   language="tsx"
 />
 
+## Close on Blur State
+
+When `closeOnBlur` is set to `true`, the `MultiSelect` accordion will close when
+focus leaves the component (the user clicks outside). In the case of a `MultiSelectGroup`,
+this means only one accordion will be open at a time.
+
+<Canvas of={MultiSelectGroupStories.closeOnBlurState} />
```

Comment:
I like this behavior and would vote to make it the default (so `closeOnBlur` set to true as the default). I (personally) get a bit annoyed when I select an option in a `MultiSelect`, open another one, but the existing one is still open. I would expect it to close but maybe that's just me.

Reply:
@EdwinGuzman I agree with your sentiment about a multiselect not closing when you move away from it, but from the NYPL UX prespective, that pattern is only applicable when the `MultiSelect` component is used in a horizonal layout filter bar.  

Additionally, I've been trying to move away from setting a positive sounding prop to `true` by default and then having to explicitly set it to `false` to negate it.  To better illustrate what I mean here, think about the labels on our form inputs.  Would we have been better off naming the label prop `hideLabel` rather than `showLabel`?  If it was `hideLabel`, then the resulting code would be simpler with `hideLabel` instead of `showLabel={false}`.